main/src/modules/reactiveLayer/behaviorManager/src/behaviorManager.cpp

```diff
@@ -75,9 +77,13 @@ bool BehaviorManager::configure(yarp::os::ResourceFinder &rf)
         Time::delay(1.0);
     }
 
-    while (!Network::connect("/ears/behavior:o", rpc_in_port.getName())) {
-        yWarning() << "Ears is not reachable";
-        yarp::os::Time::delay(0.5);
+
+    if (!rf.check("use_ears",Value("false")).asBool())
```

Comment:
Agree, will make the change. However need to check with @jypuigbo as this is also implemented in his branch `feature/upf/value`.